src/cpp/session/modules/rmarkdown/NotebookOutput.cpp

```diff
@@ -69,6 +73,8 @@ ChunkOutputType chunkOutputType(const FilePath& outputPath)
       outputType = ChunkOutputHtml;
    else if (outputPath.extensionLowerCase() == ".error")
       outputType = ChunkOutputError;
+   else if (outputPath.extensionLowerCase() == ".rdata")
```

Comment:
Since this is a valid file extension for generic "R Data" do we want to use something more data frame specific? (e.g. `.rdf`)